java/core/src/main/java/com/microsoft/bond/Serializer.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.bond;
+
+import com.microsoft.bond.helpers.ArgumentHelper;
+import com.microsoft.bond.protocol.ProtocolWriter;
+
+import java.io.IOException;
+
+/**
+ * Resposible for serialization of objects of a particular struct.
+ * @param <TStruct> the class of the serialized struct value
+ */
+public final class Serializer<TStruct extends BondSerializable> {
```

Comment:
Yes, as we have discussed and agreed on the team meeting.

---
In reply to: [123371025](https://github.com/Microsoft/bond/pull/502#discussion_r123371025) [](ancestors = 123371025)